src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java

```diff
@@ -29,7 +36,7 @@
     }
 
     private boolean isFile(Path file) {
-        return !file.toFile().isDirectory();
+        return !Files.isDirectory(file);
```

Comment:
Java 8's "Files.exists" should not be used
squid:S3725
The Files.exists method has noticeably poor performance in JDK 8, and can slow an application significantly when used to check files that don't actually exist.